packages/api/src/schema/api-schema.yaml

```diff
@@ -337,6 +337,11 @@ components:
           example: test_stream
         creatorId:
           $ref: "#/components/schemas/creator-id"
+        tags:
```

Comment:
Thom suggested renaming this to `userTags` to be extra clear. WDYT?
ref: https://www.notion.so/SHARED-Trovo-Technical-Design-Pull-Based-Ingest-8603fca9e09c4feca57da38e89c9777e?d=5966bcbf2d4641b08701d378caf7f201&pvs=4#a42563388f844e11b48c32422d14fb82